arp1/index.mjs

```diff
@@ -42,16 +42,23 @@ const setRange = (number) => {
   callBuildSequence();
 };
 
-function setTuplet(number) {
-  tuplet = number;
-  console.log('`tuplet` set to ' + number);
+function setTuplet(float) {
+  console.log(float);
+  if (float === 0) {
```

Comment:
Is it possible for the incoming tuplet message from Max to be less than 1? This just got me thinking about how UI control ranges correspond to JS variable ranges. Like if a live.dial's range is restricted to 1-16, should we also have a pattern where the incoming message is typechecked and clamped to the allowed range of values?